### ISSUE TITLE
app grid ("Show Applications") icon customization

### ISSUE THREAD

**Issue by dylnmc** [micheleg/dash-to-dock#413]
I tried to figure out how to change the "Show Applications" icon. I started to look through the code but just decided to add a features request. I figured out that it changes when you change your icon theme (in gnome-tweak-tool, e.g.). It looks like the JS just imports the icon.grid from the icon theme currently being used by GNOME.

From the next image you can see what mine looks like (my icon theme is flat-remix). I also have the elementary icon theme installed (you can see that my terminal icon for urxvt comes from the elementary icon theme), but I also like how the "Show Applications" icon in the dash-to-dock looks when I set elementary as my icon theme.

![screenshot_gnome_highlight](https://cloud.githubusercontent.com/assets/7635158/20872160/6c235388-ba6a-11e6-9c71-fabb208992d8.png)

So, in conclusion, if you could help me figure out how to change the icon for the "Show Applications" icon, that would be great. If you want to add this as a feature, too, that would be great. I just want to figure out how to change it.

Update
--
p.s. Great job with everything -- this is by far one of the most comprehensive and good-looking gnome shell extensions :+1: :)

**Comment by micheleg** [comment]
Hi,

sorry for the late reply. The icon "name" is hard-codeded in the gnome shell code: 

https://git.gnome.org/browse/gnome-shell/tree/js/ui/dash.js#n250

It then picks up whatever the theme provide. The easiest way would be to replace the icon the the theme folder. It should be this one: 

https://github.com/daniruiz/Flat-Remix/blob/master/Flat%20Remix/actions/symbolic/view-app-grid-symbolic.svg

You might even just delete it and I guess the system should pick up the default one (i don't think the elementary icon theme include this icon, but I haven't checked).

**Comment by dylnmc** [comment]
Cool thanks. No problem on the delay -- this wasn't critical by any means.

I ended up just convincing myself that I actually like the icon even though I don't. Maybe this weekend I will change it, though ... finally >:)

Thanks so much,
dylnmc
